pytest_splunk_addon/standard_lib/event_ingestors/hec_raw_ingestor.py

```diff
@@ -61,10 +61,19 @@ def ingest(self, events):
         main_event = []
         param_list = []
         for event in events:
+            if event.metadata.get("host_type") in ("plugin", None):
+                host = event.metadata["host"]
+            else:
+                host = event.key_fields["host"]
+            if event.metadata.get("timestamp_type") in ('plugin', None):
+                if not event.key_fields.get("_time"):
+                    event.key_fields['_time'] = [int(time.time())]
```

Comment:
Why `_time` is a list?

Reply:
The key_fields are expected to be a list of objects.